soundfile.py

```diff
@@ -93,10 +93,12 @@
 sf_count_t  sf_read_float    (SNDFILE *sndfile, float *ptr, sf_count_t items) ;
 sf_count_t  sf_read_double   (SNDFILE *sndfile, double *ptr, sf_count_t items) ;
 
-sf_count_t  sf_readf_short   (SNDFILE *sndfile, short *ptr, sf_count_t frames) ;
-sf_count_t  sf_readf_int     (SNDFILE *sndfile, int *ptr, sf_count_t frames) ;
-sf_count_t  sf_readf_float   (SNDFILE *sndfile, float *ptr, sf_count_t frames) ;
-sf_count_t  sf_readf_double  (SNDFILE *sndfile, double *ptr, sf_count_t frames) ;
+/* Note: The argument types were changed to void* in order to allow
+         writing bytes in SoundFile.buffer_read() */
```

Comment:
The comment isn't correct.

It's more like "... to avoid an implicit cast from 'char *' to a different pointer type in ...".

Or something like that?

You could probably even add a link to the Github issue?

I think the comment should be very clear, since it isn't really obvious why we would change the types in the declarations.